app/styles/base/_body.scss

```diff
@@ -3,6 +3,17 @@ html {
 }
 
 body {
+	background: #fff;
```

Comment:
We did have a $body-background variable (removed in this PR). I would cosnider either bringing it back or using $white here and then a corresponding _wikia-variables color var for text color.